.github/workflows/main.yml

```diff
@@ -228,7 +228,7 @@ jobs:
         run: echo "The web dir is $WEB_DIR plot dir is ${PLOT_DIR}"
 
       - name: wmass plotting
-        run: scripts/ci/run_with_singularity.sh scripts/ci/setup_and_run_python.sh scripts/plotting/makeDataMCStackPlot.py --yscale 1.3 --hists pt eta pt-eta -o $WEB_DIR -f $PLOT_DIR -p W $HIST_FILE variation --varName nominal_uncorr --varLabel MiNNLO --colors red
+        run: scripts/ci/run_with_singularity.sh scripts/ci/setup_and_run_python.sh scripts/plotting/makeDataMCStackPlot.py --yscale 1.3 --hists pt eta pt-eta --mergeGroups 'Zmumu=Zmumu,DYlowMass' 'Rare=PhotonInduced,Top,Diboson' -o $WEB_DIR -f $PLOT_DIR -p W $HIST_FILE variation --varName nominal_uncorr --varLabel MiNNLO --colors red
```

Comment:
I think these should just be defaults, right? It's annoying that they show up separately by default

Reply:
I would also group together a lot more stuff for the W